docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
@fearphage Yes, a new `iterable` matcher would be the most adequate option IMO.

Thanks for your input, I totally agree with everything you said.